app/javascript/lib/validation/shared-validation.js

```diff
@@ -93,8 +93,9 @@ export function validateUniqueCellNamesWithinFile(line, isLastLine, dataObj) {
   if (isLastLine && dataObj.duplicateCellNames.size > 0) {
     const nameTxt = (dataObj.duplicateCellNames.size > 1) ? 'duplicates' : 'duplicate'
     const dupString = [...dataObj.duplicateCellNames].slice(0, 10).join(', ')
-    const msg = `Cell names must be unique within a file. 
-      ${dataObj.duplicateCellNames.size} ${nameTxt} found, including: ${dupString}`
+    const msg =
```

Comment:
Why are we getting rid of multiline strings?

Reply:
It made the related test fail.  Multiline template string get evaluated as having multiple lines, yet remain shown as one line in the web UI.

```
  ● Client-side file validation › catches duplicate cell names in cluster file

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 2

    - Cell names must be unique within a file. 1 duplicate found, including: CELL_0001
    + Cell names must be unique within a file. 
    +       1 duplicate found, including: CELL_0001

      129 |     expect(errors).toHaveLength(1)
      130 |     expect(errors[0][1]).toEqual('content:duplicate:cells-within-file')
    > 131 |     expect(errors[0][2]).toEqual('Cell names must be unique within a file. 1 duplicate found, including: CELL_0001')
          |                          ^
      132 |   })
      133 |
      134 |   it('catches duplicate cell names in expression matrix file', async () => {

      at Object.<anonymous> (test/js/lib/validate-file-content.test.js:131:26)

```